CMakeLists.txt

```diff
@@ -166,9 +166,9 @@ if (NJOBS)
   set(CMAKE_JOB_POOL_PRECOMPILE_HEADER custom)
 endif()
 
-# GCC 8.1 is required to support <charconv> header inclusion in base/string_utils.hpp, otherwise 7.0 is sufficient
-if (CMAKE_CXX_COMPILER_ID MATCHES "GNU" AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 8.1)
-  message(FATAL_ERROR "Minimum supported g++ version is 8.1 yours is ${CMAKE_CXX_COMPILER_VERSION}")
+# GCC 8.1 is required to support <charconv> header inclusion in base/string_utils.hpp
```

Comment:
```suggestion
# GCC 10.0 is required to support <charconv> header inclusion in base/string_utils.hpp
```

Is GCC version mentioned somewhere in the Linux build docs? Should we mention it explicitly there too?

Reply:
> Should we mention it explicitly there too?

Yes, but IMHO that can also go into a separate PR.